docker/tools/Dockerfile

```diff
@@ -22,6 +22,8 @@ COPY . /libra
 
 RUN ./docker/build-common.sh
 
+RUN curl --proto '=https' --tlsv1.2 -sSfL https://aka.ms/downloadazcopy-v10-linux | tar xzv
```

Comment:
@mgorven  Happy with this? Or shall I create separate WORKDIR?

Reply:
Do this directly in the production image since we don't have build artifacts.

    cd /usr/local/bin/ && curl --proto '=https' --tlsv1.2 -sSfL https://aka.ms/downloadazcopy-v10-linux | tar -vz -x '*azcopy' --wildcards --strip-components=1

Reply:
Do this directly in the production image, since there aren't build artifacts to deal with.

    RUN cd /usr/local/bin && curl --proto '=https' --tlsv1.2 -sSfL https://aka.ms/downloadazcopy-v10-linux | tar -vz -x '*azcopy' --wildcards --strip-components=1